modules/getting-started/proc-techdocs-configure-odf-helm.adoc

```diff
@@ -0,0 +1,19 @@
+[id="proc-techdocs-configure-odf-helm_{context}"]
+
+= Adding access information to the {product} Helm chart
+
+Add access information for the `ObjectBucket` to the {product} Helm chart configuration.
```

Comment:
We should clarify that this is only required if Helm Chart was used to deploy RHDH. And if they are using RHDH Operator than they should follow the next section and skip this one.